src/main/java/RidesRequestHandler.java

```diff
@@ -1,12 +1,66 @@
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
-import main.java.models.AddInfoRequest;
+import dynamo.RidesDao;
+import models.ClientRequest;
 
-public class RidesRequestHandler implements RequestHandler<AddInfoRequest, String> {
+
+public class RidesRequestHandler implements RequestHandler<ClientRequest, String> {
+
+    private RidesDao ridesDao = new RidesDao();
 
     @Override
-    public String handleRequest(AddInfoRequest request, Context context) {
-        return "foo";
+    public String handleRequest(ClientRequest request, Context context) {
+
+        switch(request.getRequestType()) {
+            case ADD:
+                addInfo(request);
+                break;
+            case DELETE:
+                deleteInfo(request);
+                break;
+            case EDIT:
+                editInfo(request);
+                break;
+            case CANCEL:
+                cancelInfo(request);
+                break;
+            case RETRIEVE:
+                retrieveInfo(request);
+                break;
+            default:
+                // panic
+        }
+
+        return "";
+    }
+
+    private int addInfo (ClientRequest request)

```

Comment:
Can you please add some comments above each function signature for what these functions are supposed to do! Just do your best and we can formalize them a bit more later on.